spec/factories/users.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do 
   factory :user do 
-    sequence(:name) {|n| "Name #{n}"}
-    sequence(:email) {|n| "test#{n}@example.com"}
+    name FFaker::Name.name
```

Comment:
Doesn't this mean that every factory-generated user will have the same fake name?